test/demo/embed.html

```diff
@@ -0,0 +1,33 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CartoWall Climbing Topo</title>
```

Comment:
As an amateur climber, I had to look up that website :)
Good job there! Would you like to add it to http://openseadragon.github.io/examples/in-the-wild/ ?